test/general/x25519.js

```diff
@@ -396,7 +396,7 @@ function omnibus() {
       const primaryKey = hi.keyPacket;
       const subkey = hi.subkeys[0];
       expect(hi.getAlgorithmInfo().curve).to.equal('ed25519');
-      expect(hi.getAlgorithmInfo().algorithm).to.equal('eddsa');
+      expect(hi.getAlgorithmInfo().algorithm).to.equal('ed25519Legacy');
```

Comment:
@twiss: I know we already introduced for `ed25519Legacy` on v5, but I was thinking that `eddsaLegacy` would be a better name than `ed25519Legacy`, because
1) the reference with the old name is clear even if someone is not familiar with what ed25519 is, and
2) `getAlgorithmInfo()` (and some errors) will now refer to `algo = ed25519Legacy` and `curve = ed25519`, which I think it's ugly, confusing and ultimately redundant.
What do you think?

Reply:
Yeah, I agree; actually in the crypto refresh the name is also [EdDSALegacy](https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-10.html#name-public-key-algorithms), and then the curve name is [Ed25519Legacy](https://www.ietf.org/archive/id/draft-ietf-openpgp-crypto-refresh-10.html#name-ecc-curves-for-openpgp), we should do the same (uncapitalized) here.

Reply:
Should i change v5 too? It might be breaking if people started using those names

Reply:
Probably best to change it in v6 only

Reply:
Should still add an additional alias in v5, to ease the transition? 😅

Reply:
I did at it to the last v5 release, so I'll have to also add `curve.curve25519Legacy` over `curve.x25519Legacy`, but it can be done after merging the outstanding v6 sub-PRs.